styles/_theme.scss

```diff
@@ -534,4 +534,14 @@ input[type=range][class~="clrS"] {
       background: linear-gradient(to right, transparent, $primary 50%);
     }
   }
+}
+
+.btnFlx {
+  // @include borderColor(transparent, transparent);
+  border: none;
```

Comment:
Take this out, you don't want to override the width and style from the buttons stylesheet. Instead there should be a border reset there.